study-builder/studies/pancan/i18n/en.conf

```diff
@@ -466,7 +466,7 @@
     "s1_participation_body1": "After signing a consent form, you will be asked to complete surveys about your child’s experience with cancer. Additional survey(s) may become available during the course of this research study that you can choose to complete.",
     "s1_participation_body2": "We will ask that you send a sample of your child’s saliva to us in a pre-stamped package that we will provide.",
     "s1_participation_body3": "We will ask you to complete a medical release form so that we are able to request your child’s medical records. If we do so, we will take care of obtaining copies of your child’s records from the hospitals or centers where your child receives their medical care. If needed, we may contact you to ask if you would be willing to sign additional authorization forms or documents that certain hospitals or centers may require in order to share copies of your child’s medical records with us.",
-    "s1_participation_body4": "If you choose to share tissue specimens with us, we may also obtain small amounts of your child’s stored cancer samples from hospitals or centers where your child receives their care.",
+    "s1_participation_body4": "If you choose to s"The Osteosarcoma Project" is a patient-driven movement that empowershare tissue specimens with us, we may also obtain small amounts of your child’s stored cancer samples from hospitals or centers where your child receives their care.",
```

Comment:
check if this variable is used somewhere else, to make sure your patch is not updating other place accidentally

Reply:
This variable is not used other place.